.env.example

```diff
@@ -1,6 +1,11 @@
 RDV_SOLIDARITES_URL=http://localhost:5000
 HOST=http://localhost:8000
 
+# ActiveRecord encryption
```

Comment:
nitpick: j'aurais peut-être rajouté un commentaire avec la commande pour générer les clés en local

Reply:
peut-être aussi recopier ici le warning de la doc : `These generated values are 32 bytes in length. If you generate these yourself, the minimum lengths you should use are 12 bytes for the primary key (this will be used to derive the AES 32 bytes key) and 20 bytes for the salt.`